.travis.yml

```diff
@@ -1,21 +1,12 @@
-language: scala
+language: java
 sudo: false
 jdk:
   - oraclejdk8
 script:
-  - sbt +test
-cache:
```

Comment:
It might be a good idea to retain a cache to speed the build: https://docs.travis-ci.com/user/caching#Arbitrary-directories